standard/ERCs/erc-6900.md

```diff
@@ -683,6 +683,12 @@ The `executeFromModule` function MUST allow modules to call execution functions
 
 The `executeFromModuleExternal` function MUST allow modules to call external addresses as specified by its parameters on behalf of the modular account. If the calling module's manifest did not explicitly allow the external call within `permittedExternalCalls` at the time of installation, execution MUST revert.
 
+## Extension
```

Comment:
I don't think we can put this as a top-level section (with "`##`"), because it's not a part of the standard format for ERCs. This should fit into either the specification or rationale sections.

https://github.com/ethereum/ERCs/blob/master/erc-template.md?plain=1
https://eips.ethereum.org/EIPS/eip-1

Reply:
Good catch. Meant to put it as part of the spec. Updated.